tests/foreman/installer/test_installer.py

```diff
@@ -1361,6 +1378,7 @@ def sat_non_default_install(module_sat_ready_rhels):
 
 @pytest.mark.e2e
 @pytest.mark.tier1
+@pytest.mark.pit
```

Comment:
Great to see this inclusion, @rmynar! Please get in touch with me after this PR is merged. I would like to do a test execution of the PIT pipeline to see how the test behaves in it.